jest.config.js

```diff
@@ -136,7 +139,7 @@ const testReactComponents = inPackage => ({
         '@magento/venia-drivers':
             '<rootDir>/packages/venia-ui/lib/drivers/index.js'
     },
-    moduleFileExtensions: ['ee.js', 'ce.js', 'js', 'json', 'jsx', 'node'],
+    moduleFileExtensions: [...defaults.moduleFileExtensions, 'ee.js', 'ce.js'],
```

Comment:
I made this change. Is `moduleFileExtensions` in order of priority? This is jest config, not webpack: https://jestjs.io/docs/en/webpack#configuring-jest-to-find-our-files

Reply:
As far as I remember yes. For instance, if you have `component.js` and `component.json` in the same folder you try to import `component` since `.js` has higher priority, it will be picked over `.json` file. And we follow the same array convention in webpack configuration as well. It is beneficial if these 2 stay in sync. `Webpack` uses a similar import configuration while building which jest uses while testing.

Reply:
You're right! We should find a way to sync these! We should export the module resolution array from some central location. @larsroettig if you can do that, would be awesome, otherwise we can just revert this change back to how it was before I modified it.